### ISSUE TITLE
Spontaneous Casters can no longer charge staves expending spell slots

### ISSUE THREAD

**Issue by blacktrench** [jessev14/pf2e-staves#33]
When I try to charge staves with spontaneous casters I only get the option to charge the staff to the highest spell slot not to expend slots, it functions normally for prepared casters. I've disabled all modules except staves and you will be able to see my foundry vtt version and pf2e version (I am on 10, have not upgraded to 11 yet).

Thanks!

![image](https://github.com/jessev14/pf2e-staves/assets/24504912/6de208e2-5a1d-4681-9cf8-3a368ea2b9bd)

**Comment by this-gavagai** [comment]
Hmm...Isn't that the way it is supposed to work? I thought adding extra charges was only for prepared spell casters. Spontaneous casters have a different mechanic to extend charges by expending a slot during use.

https://2e.aonprd.com/Rules.aspx?ID=748

**Comment by blacktrench** [comment]
Yeah I might have been mistaken but I assumed like sacrificing prepared spell you could have the mechanics of spending 1 charge + spell LVL using the charge option.

So you could click the charge button and get an option to sacrifice a spell slot to reduce the cost and not have to manually change it.

If this was never included or intended that is my bad and instead of a bug it would be a feature request lol.
